contracts/FeesAndBootstrapRewards.sol

```diff
@@ -105,7 +108,9 @@ contract FeesAndBootstrapRewards is IFeesAndBootstrapRewards, ManagedContract {
 
         uint256 amount = feesAndBootstrap[guardian].feeBalance;
         feesAndBootstrap[guardian].feeBalance = 0;
-        emit FeesWithdrawn(guardian, amount);
+        feesAndBootstrap[guardian].withdrawnFees = feesAndBootstrap[guardian].withdrawnFees.add(amount);
```

Comment:
can assign the updated withdrawnFees in a memory variable

Reply:
Done